package.json

```diff
@@ -18,13 +18,14 @@
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
-    "test": "node test.js",
+    "test": "sh setup.sh",
```

Comment:
The original `setup.sh` is used to set up the docker image for cloudant. it's not for testing. It would be good to keep it this way.

Reply:
Thanks for the early review. And sure. But then it will have two `sh` files that are mostly the same if that's okay.